masschallenge/impact-api
# AC-6074: Entrants should not be able to see mentors on `/directory/`

**Changes introduced in [AC-6074](https://masschallenge.atlassian.net/browse/AC-6074)**:
- introduce the `AC-6023-fix` that ensures we correctly assume the alumni program is active
- add Directory permissions that stop entrants from accessing the directory

**How to test**:
- Ensure impact-api and the directory are both on `AC-6074`
- register as an entrepreneur on `accelerate`
- visit the directory with that user
- you should get a 403

*Previously entrants were able to access the directory
